person.rb

```diff
@@ -1,14 +1,16 @@
 require './corrector'
+require './rental'
 
 class Person
-  attr_accessor :id, :name, :age
+  attr_accessor :id, :name, :age, :rentals
```

Comment:
- [OPTIONAL] Since the id is auto-generated in the initialize method. You may make it only readable.
```suggestion
  attr_accessor :name, :age, :rentals
  attr_reader :id
```

Reply:
Thanks for the suggestion. I will update it.